docs/contracts/overview.md

```diff
@@ -13,12 +13,15 @@ are based on two NFT standards,
 ## Contracts description
 
 Main contracts:  
-* `BosonRouter`: user interface of Boson Protocol  
-* `Cashier`: escrow and funds management  
-* `ERC1155ERC721`: token factory  
-* `FundLimitsOracle`: restrictions on the allowed escrowed amounts  
-* `VoucherKernel`: main business logic  
-* `UsingHelpers`: common utilities  
+* `BosonRouter`: User interface of Boson Protocol  
+* `Cashier`: Escrow and funds management  
+* `ERC1155ERC721`: Token factory  
+* `TokenRegistry`: Restrictions on the allowed escrowed amounts and registry of token wrappers
+* `VoucherKernel`: Main business logic  
+* `UsingHelpers`: Common utilities
+* `DAITokenWrapper`: Provides a uniform interface for calling the `permit `function on the DAI token
+* `ERC1155NonTransferable`: Non-transferrable token, the possession of which allows a buyer to commit to redeem an item following a special quest or campaign.
+* `Gate`: Handles the interaction between Boson router and ERC1155NonTransferable
```

Comment:
allowing user's of the protocol to gate access to a given VoucherSet by requiring that any address attempting to commit to an item in the given VoucherSet have a given ERC721 before they access.

Reply:
Changed, but substituted ERC1155 for ERC721.  Will be included in next commit.